src/annotations/components/annotationForm/AnnotationForm.tsx

```diff
@@ -29,24 +29,45 @@ type AnnotationType = 'point' | 'range'
 
 interface Props {
   startTime: string
+  endTime?: string
   title: 'Edit' | 'Add'
   type: AnnotationType
   onSubmit: (Annotation) => void
   onClose: () => void
 }
 
+export const checkAnnotationFormValidity = (
+  annotationType: string,
+  message: string,
+  startTime: any,
+  endTime: any
+) => {
+  const firstPart = message.length && startTime
```

Comment:
```suggestion
  const isValidPointAnnotation = message.length && startTime
```

`firstPart` seems like a stored value, not a boolean. Even reading the line that created the value, it wasn't super clear to me that it's a boolean. You don't have to call it `isValidPointAnnotation`, but that's the concept we're trying to convey, right?